src/App.js

```diff
@@ -75,7 +75,7 @@ class App extends Component {
   render() {
     console.log(this.state.cats)
     return(
-      <Router>
+      <Router class="router">
```

Comment:
In React the modification on a tag is done with a `className` since class is already a keyword in JavaScript

Reply:
Thanks for catching that. We were just experimenting and found a different way to solve our problem.